src/main/resources/templates/db_scripts/create_failed_ledger_rollover_po_line_table.sql

```diff
@@ -0,0 +1,14 @@
+-- Creates table to store failures during rollover
+CREATE TABLE IF NOT EXISTS ${myuniversity}_${mymodule}.failed_ledger_rollover_po_line
+(
+  id uuid NOT NULL PRIMARY KEY,
+  rollover_id uuid,
+  ledger_id uuid,
+  po_line_id uuid,
+  request_body text,
+  response_body text,
+  status_code text,
+  workflow_status text
+);
+
+CREATE INDEX IF NOT EXISTS rollover_id_idx ON ${myuniversity}_${mymodule}.failed_ledger_rollover_po_line USING btree (rollover_id);
```

Comment:
I believe I you have meant pattern 'table_name'_'column_name'_idx
Also pretty long index names potentially could cause issues because of length limits. Current lenght is 43 characters of 63 allowed, so lets hope that will never be concatenatedd with something else